sci-mathematics/rstudio/rstudio-1.4.1106.ebuild

```
@@ -0,0 +1,97 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake qmake-utils
+
+QT_VER=5.12.6
+QT_SLOT=5
+
+DESCRIPTION="IDE for the R language"
+HOMEPAGE="
+	https://rstudio.com
+	https://github.com/rstudio/rstudio/"
+SRC_URI="
+	https://github.com/rstudio/rstudio/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://s3.amazonaws.com/rstudio-dictionaries/core-dictionaries.zip
```

I'd suggest versioning this somehow in case it gets changed in future, in case we have a few versions in tree (and so the file on user machines is labelled), e.g.

```suggestion
	https://s3.amazonaws.com/rstudio-dictionaries/core-dictionaries -> core-dictionaries-1.zip
```

---

I had a similar though actually, just didn't let the thought fully play out in my head 🤣

---

Added `${P}` to the filename.